### PR TITLE
lynx: Update to 2.9.0dev.11

### PR DESCRIPTION
**Pull request by ThomasDickey**
#### Description

Update lynx to current version.   Besides the assorted bug/security fixes,
this modifies the port to use compression in lynx, matching the most common
usage.

###### Type(s)

- [X] bugfix
- [X] enhancement
- [X] security fix

###### Tested on
macOS 12.6.1 21G217 x86_64
Xcode 14.2 14C18

###### Verification 
Have you

- [X] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [X] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [X] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [X] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [X] tried a full install with `sudo port -vst install`?
- [X] tested basic functionality of all binary files?
- [X] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

File: www/lynx/Portfile [hunk 1]
```diff
@@ -3,8 +3,8 @@
 PortSystem          1.0
 
 name                lynx
-set milestone       2.8.9
-version             ${milestone}rel.1
+set milestone       2.9.0
+version             ${milestone}dev.11
```
Comment (reneeotten) [inline www/lynx/Portfile:7]: this name suggest it's actually not a released version but a development version. Typically one would only update a port to its latest, stable release.

**Comment by ThomasDickey** [comment]
See [this](https://invisible-island.net/lynx/lynx-develop.html).

**Comment by ThomasDickey** [comment]
lack of response.

**Comment by reneeotten** [comment]
really... you expect a response in 12h and otherwise you close a PR? I am traveling and had no time to look at your response to my IMHO reasonable review comment. 

I am happy to take another look but if am not going to spend my time if you're not willing to be a little patient...

**Pull request closed without merging**